### PR TITLE
fix FBXGeometry::convexHullContains crash

### PR DESCRIPTION
**Pull request by sethalves**
- possible fix for FBXGeometry::convexHullContains crash

### CODE REVIEW

**Comment by sethalves** [comment]
https://app.asana.com/0/26485730907942/101905459350876

**Comment by hifi-gustavo** [comment]
Build console output for commit: cea97aa317389b538c345725c810bea44762c75b available at: https://gist.github.com/3db95b73ad4e43e53cf3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7381.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7381.dmg

File: libraries/fbx/src/FBXReader.cpp [hunk 1]
```diff
@@ -76,7 +76,7 @@ bool FBXGeometry::convexHullContains(const glm::vec3& point) const {
 
     auto checkEachPrimitive = [=](FBXMesh& mesh, QVector<int> indices, int primitiveSize) -> bool {
         // Check whether the point is "behind" all the primitives.
-        for (int j = 0; j < indices.size(); j += primitiveSize) {
+        for (int j = 0; j < indices.size() - 2; j += primitiveSize) {
```
Comment (ZappoMan) [inline libraries/fbx/src/FBXReader.cpp]: what??? magic... why -2?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 87dbec8974ca4cecc61fc7d0f67f07b658deffb9 available at: https://gist.github.com/3db95b73ad4e43e53cf3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7381.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7381.dmg

**Comment by cozza13** [comment]
I have been smoke testng this build for 30 minutes, while doing edits. Did not crash. QA pass

**Pull request merged**